lib/arel/middleware/postgresql_adapter.rb

```diff
@@ -38,20 +38,20 @@ def query(sql, name = nil)
         end
       end
 
-      def exec_no_cache(sql, name, binds)
+      def exec_no_cache(sql, name, binds, async: false)
```

Comment:
the default in Rails is false?

Reply:
`vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.4.3/lib/active_record/connection_adapters/postgresql_adapter.rb`

```ruby

        def exec_no_cache(sql, name, binds, async: false)
          materialize_transactions
          mark_transaction_written_if_write(sql)

          # make sure we carry over any changes to ActiveRecord.default_timezone that have been
          # made since we established the connection
          update_typemap_for_default_timezone

          type_casted_binds = type_casted_binds(binds)
          log(sql, name, binds, type_casted_binds, async: async) do
            ActiveSupport::Dependencies.interlock.permit_concurrent_loads do
              @connection.exec_params(sql, type_casted_binds)
            end
          end
        end
```